data-source/blocks/individual/identity-and-health/language.jsonnet

```diff
@@ -1,110 +1,86 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import '../../../lib/rules.libsonnet';
 
-local question(proxyOptions, regionOptions) = {
+local question(title, definitionContent, regionOptions) = {
```

Comment:
Minor, `regionOptions` is only ever one item, so should be `regionOption` (or possibly `regionalOption`).